server/sonar-process/src/main/java/org/sonar/process/ProcessEntryPoint.java

```diff
@@ -21,6 +21,7 @@
 
 import java.io.File;
 import org.slf4j.LoggerFactory;
+import sun.misc.VMSupport;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of Sun classes by ones from the Java API. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1191)